src/applications/static-pages/facilities/buildVamcOperatingStatusProps.js

```diff
@@ -0,0 +1,30 @@
+export const buildVamcOperatingStatusProps = attrs => {
+  const { opStatus, opStatusLabel, opStatusExtra } = attrs;
+  let iconType;
+
+  if (opStatus === 'normal') {
+    return;
+  }
+
+  switch (opStatus) {
+    case 'limited':
```

Comment:
Should map to cases of the field_status_id to the values identified in the storybook component.   That way it is not specific to operating statuses which are not related.